infrastructure/smart-contracts/contracts/upgrades/referee/Referee10.sol

```diff
@@ -1106,6 +1112,21 @@ contract Referee10 is Initializable, AccessControlEnumerableUpgradeable {
 		}
     }
 
+    function updateBulkSubmissionOnTransfer(address from, address to, uint256 keyId) external {
+        if(msg.sender != nodeLicenseAddress) revert CallerNotAuthorized();
+
+        uint256 currentChallenge = challengeCounter == 0 ? 0 : challengeCounter - 1;  
```

Comment:
I assume this is to have the test suite nor require an initial challenge to exist ?

Reply:
Yes, that is correct.